common/CMakeLists.txt

```diff
@@ -138,3 +138,13 @@ target_include_directories(identity_parser_test PRIVATE ${CMAKE_SOURCE_DIR})
 gtest_discover_tests(identity_parser_test)
 add_dependencies(check identity_parser_test)
 
+
+add_library(common_conf STATIC conf/conf.cpp conf/paths.cpp)
+target_link_libraries(common_conf PUBLIC crossplatform_compiler_flags)
+target_include_directories(common_conf PRIVATE ../ ${CMAKE_BINARY_DIR})
```

Comment:
Use `${CMAKE_SOURCE_DIR}` instead of `../`. The latter has broken for me in some circumstances.

Reply:
Will fix it in other places so that I don't take one of them as a template again next time.